lexicons/app/bsky/label/defs.json

```diff
@@ -0,0 +1,103 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.label.defs",
+  "defs": {
+    "labelerView": {
+      "type": "object",
+      "required": ["uri", "cid", "did", "creator", "displayName", "indexedAt"],
+      "properties": {
+        "uri": { "type": "string", "format": "at-uri" },
+        "cid": { "type": "string", "format": "cid" },
+        "did": { "type": "string", "format": "did" },
+        "creator": { "type": "ref", "ref": "app.bsky.actor.defs#profileView" },
+        "displayName": { "type": "string" },
+        "description": {
+          "type": "string",
+          "maxGraphemes": 300,
+          "maxLength": 3000
+        },
+        "descriptionFacets": {
+          "type": "array",
+          "items": { "type": "ref", "ref": "app.bsky.richtext.facet" }
+        },
+        "avatar": { "type": "string" },
+        "policies": {"type": "ref", "ref": "app.bsky.label.defs#labelerPolicies"},
+        "likeCount": { "type": "integer", "minimum": 0 },
+        "viewer": { "type": "ref", "ref": "#labelerViewerState" },
+        "indexedAt": { "type": "string", "format": "datetime" }
+      }
+    },
+    "labelerViewerState": {
+      "type": "object",
+      "properties": {
+        "like": { "type": "string", "format": "at-uri" }
+      }
+    },
+    "labelerPolicies": {
+      "type": "object",
+      "required": ["reportReasons", "labelValues"],
+      "properties": {
+        "description": {
+          "type": "string",
+          "maxGraphemes": 10000,
+          "maxLength": 100000
+        },
+        "descriptionFacets": {
+          "type": "array",
+          "items": { "type": "ref", "ref": "app.bsky.richtext.facet" }
+        },
+        "reportReasons": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "knownValues": [
+              "com.atproto.moderation.defs#reasonSpam",
+              "com.atproto.moderation.defs#reasonViolation",
+              "com.atproto.moderation.defs#reasonMisleading",
+              "com.atproto.moderation.defs#reasonSexual",
+              "com.atproto.moderation.defs#reasonRude",
+              "com.atproto.moderation.defs#reasonOther",
+              "com.atproto.moderation.defs#reasonAppeal"
+            ]
+          }
+        },
+        "labelValues": {
```

Comment:
may wanna pull this out into it's own def as well - could even re-use the def in the `label` "record" type

Reply:
+1 if we can do refs for these kinds of values, my memory was that refs only worked for objects and arrays

Reply:
we've been using them for strings like this 👍 

you can see an example of this in `com.atproto.admin.defs#modEventReport`